src/index.js

```diff
@@ -4,3 +4,4 @@ export elementType from './elementType';
 export keyOf from './keyOf';
 export singlePropFrom from './singlePropFrom';
 export all from './all';
+export deprecated from './deprecated';
```

Comment:
I don't think it makes sense to expose this publicly, its really only useful to us as project devs.